OasisDEX/oasis-borrow
# Remove `WithChildren` from codebase

# [Remove `WithChildren` from codebase](https://shortcutLink)

![image](https://github.com/OasisDEX/oasis-borrow/assets/16230404/f80b9e28-34b3-4e2b-8a92-774d9b2b1496)
  
## Changes 👷‍♀️

- Replaced all `WithChildren` with native to React `PropsWithChildren`.
  
## How to test 🧪

As this is only typescript change, no need to be tested - if build passes everything should work.
